packages/web-ui/src/ds/molecules/DiffViewer/Editor/useMonacoSetup.ts

```diff
@@ -1,16 +1,13 @@
 import { useCallback, useEffect, useRef } from 'react'
 
-import { loader, type Monaco } from '@monaco-editor/react'
-import * as monaco from 'monaco-editor'
+import { type Monaco } from '@monaco-editor/react'
 
 import {
   themeRules,
   tokenizer,
   useThemeColors,
 } from '../../../../lib/monacoEditor/language'
 
-loader.config({ monaco })
```

Comment:
What is this used for? react wrapper already loads monaco and this was the cause of the whole web worker not loading drama